features/networking/metrics.feature

```diff
@@ -84,7 +84,13 @@ Feature: SDN/OVN metrics related networking scenarios
     And evaluation of `cb.ovn_master_metrics_ep_ip + ':' +cb.ovn_master_metrics_ep_port` is stored in the :ovn_master_metrics_ep clipboard
     
     And evaluation of `endpoints('ovn-kubernetes-node').subsets.first.addresses.first.ip.to_s` is stored in the :ovn_node_metrics_ep_ip clipboard
-    And evaluation of `endpoints('ovn-kubernetes-node').subsets.first.ports.first.port.to_s` is stored in the :ovn_node_metrics_ep_port clipboard
+    When I run the :get client command with:
+      | resource      | ep                                                      |
+      | n             | openshift-ovn-kubernetes                                |
+      | resource_name | ovn-kubernetes-node                                     |
+      | o             | jsonpath={.subsets[*].ports[?(@.name=="metrics")].port} |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :ovn_node_metrics_ep_port clipboard
```

Comment:
We can do jsonpath if that is more familiar.

A ruby equivalent would be
```ruby
    And evaluation of `endpoints('ovn-kubernetes-node').subsets.flat_map{ |i| i.ports }.select{ |p| p.name == "metrics" }.first.port.to_s` is stored in the :ovn_node_metrics_ep_port clipboard
```

Reply:
@anuragthehatter I think we prefer the ruby approach...  You can even add a method to make it easier on the eyes.

Reply:
@rbbratta @pruan-rht Oops..Yea could have used ruby equivalent as I used same in other steps as well. Anyways used it now..heh. Thanks. and re-ran passed